.github/workflows/scripts/rubygems-publish.rb

```diff
@@ -21,7 +22,7 @@
 if $?.to_i.zero?
   puts "#{gem_filename} successfully pushed to rubygems.org!"
 else
-  if result =~ /Repushing of gem versions is not allowed/
+  if result.include?('Repushing of gem versions is not allowed')
```

Comment:
Rubocop was mad about this file for some reason, so I had to make these changes to get it to commit and run the tests.

Reply:
Using `=~` produces a fresh `Regexp.last_match` object to store the overall matched text as well as any individual captures. Given that we don't want to do anything with the matched check and simply verify with a boolean that there is a match, using `#match?` or `#include?` seems more efficient. Good change!